cmd/cli/app/verify.go

```diff
@@ -35,6 +35,17 @@ var verifyCmd = &cobra.Command{
 	Use:   "verify",
 	Short: "Rekor verify command",
 	Long:  `Verifies a signature and checks that it exists in the transparency log`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		// these are bound here so that they are not overwritten by other commands
+		if err := viper.BindPFlags(cmd.Flags()); err != nil {
+			log.Logger.Fatal("Error initializing cmd line args: ", err)
+		}
+		if err := validateRekorServerURL(); err != nil {
```

Comment:
Maybe we could handle this in a custom flag type instead of needing to put it in the PreRun everywhere?

https://golang.org/pkg/flag/#Value

Reply:
+1 on the validation change; however, if we want consistent flags between sub commands we need the BindPFlags call in prerun so that only the command being executed gets the values bound to them.

Reply:
Makes sense. I'll handle the validation part.